### ISSUE TITLE
use system umask instead of hard-coding file+dir permissions

### ISSUE THREAD

**Issue by cweiske** [pyrus/Pyrus#117]
Pyrus often uses `mkdir($foo, 0755)`, which is not always wanted.

In our company, we have many people using the same pyrus installation on the same machine with different users. We need 0775 as permissions.

Pyrus should either use the umask, or have a configuration option to modify.

**Comment by phaidros7** [comment]
I agree, explicit 0755 prevents ACLs from working: with default ACLs for e.g. a group all files inherit those, but with explicit 0755, or better g-w the POSIX permissions are stronger than the ACLs and the whole multiuser problem is unsolved again :/

I second cweiske's proposal: either umask or config option!
